book/src/reference/config.md

```diff
@@ -38,6 +38,7 @@ We're currently in the middle of [a major config migration](https://github.com/a
     * [`all-features`](#all-features)
     * [`default-features`](#default-features)
     * [`features`](#features)
+    * [`minimum-glibc-version`](#minimum-glibc-version)
```

Comment:
i might shorten this to `min` to reduce possibility of mispellings

Reply:
I'm okay with this suggestion. :+1:

Reply:
I just realized all the existing code for auto-detecting glibc stuff was called `min_glibc_version`, so shortening it improves the internal consistency of the codebase, too.

Reply:
Since I had to change the field names for some of the v1 config to make this work, I changed all of it to avoid weird mismatches between "minimum" and "min" being scattered all over the place.